tests/common/fixtures/services.py

```diff
@@ -0,0 +1,25 @@
+from unittest.mock import create_autospec
+
+import pytest
+
+from h.services.search_index import SearchIndexService
+
+# Allow people to mass import everything without getting things we don't
+# mean them to like pytest.
+__all__ = ("service_mocker", "search_index")
+
+
+@pytest.fixture
+def service_mocker(pyramid_config):
+    def service_mocker(class_, name):
+        service = create_autospec(class_, instance=True)
```

Comment:
```suggestion
        service = create_autospec(class_, instance=True, spec_set=True)
```

Same as it's done for [the service mocks in lms](https://github.com/hypothesis/lms/blob/7e8ea35d79d98a2660aef17312ddcf21190279c1/tests/unit/conftest.py#L157-L256). See <https://docs.python.org/3/library/unittest.mock.html#unittest.mock.create_autospec> for what `spec_set` does.

I think `mock_service()` might be a better name than `service_mocker()`? The function returns a mock of a given service, it doesn't return a "service mocker".